### PR TITLE
Journal week 2 to master

### PR DESCRIPTION
**Pull request by s4dmw**

### CODE REVIEW

**Comment by jeffworley** [comment]
Good luck with that URL health issue.  lol.  I like that humorous play on topics and look forward to the additional information you pack in tomorrow. :+1:

**Comment by ericksonk** [comment]
Sweet puns :+1: Very descriptive and a good overview of the week!

File: journal-week-2.md [hunk 1]
```diff
@@ -1,34 +1,15 @@
-#### Journal, Week 2 (Rough Draft)
+#### Tears On My Keyboard: A Journal of Week 2
```
Comment (kellymurray) [inline journal-week-2.md:1]: :laughing:

File: journal-week-2.md [hunk 2]
```diff
@@ -1,34 +1,15 @@
...
+Adding to the pain, the week was full of conflicts. Merge conflicts to be exact. We spent countless hours, days, weeks, resolving, pushing, pulling, rinsing, repeating. By the end of week two, I was completely exhausted but my GitHub house was in order.
 
-Describe the MVC pattern of software design, as you understand it.
-Explain the principle of Encapsulation and give some examples from your own code.
-How does scoping work in JavaScript programs? How do the function and var keywords affect it?
+The flow of our control was way off this week. One might say it was altered. We spent some time looping, functioning, conditioning, and operating on our precedence to get the control back.
+
+The object of dis-array might seem unclear but if you dig deeper, there are many properties that can be accessed in their own ways. If you have the right keys and the proper values, you too can unlock the dictionaries and discover a whole world.
+
+One glimmer of hope for the week was getting to indulge in my love of all things coffee. Never in my life have I encountered as much mocha and chai being used in java...script.
```
Comment (kellymurray) [inline journal-week-2.md:11]: Love your entry. :)

**Pull request merged**